src/pages/index.tsx

```diff
@@ -35,6 +36,11 @@ import useTinyCount from '../useTinyCount'
 import { update } from 'lodash'
 
 const INTRO_SEEN_KEY = 'intro_seen'
+const adminUsers: string[] = [
+  '@wokebot:woke.net',
+  '@chromakode:woke.net',
+  '@tree:woke.net'
+]
```

Comment:
Let's hold off from differentiating admins right now and split this into a separate project / discussion. I'm not sure if/how we'd like to display this, and the status should be sourced from the matrix user "power level" (>=50 means the user is a mod, 100 means the user is an admin)